include/mbedtls/ecdh.h

```diff
@@ -63,6 +106,17 @@ typedef struct mbedtls_ecdh_context
     mbedtls_ecp_point Vi;    /*!< The blinding value. */
     mbedtls_ecp_point Vf;    /*!< The unblinding value. */
     mbedtls_mpi _d;          /*!< The previous \p d. */
+#else
+    uint8_t point_format;    /*!< The format of point export in TLS messages as
+                                  defined in RFC 4492. */
+    mbedtls_ecp_group_id grp;/*!< The elliptic curve used. */
```

Comment:
Minor: I'd prefer `gid` or something else than `grp` here to avoid confusion. Usually `grp` means an `mbedtls_ecp_group`.